jsk_fetch_robot/jsk_fetch_startup/apps/volume_zero/volume_zero.app

```diff
@@ -0,0 +1,6 @@
+display: Fetch Volume Zero
+description: Fetch speaks zero volume
+platform: fetch
```

Comment:
we can use `platform: all`.
with that, we can move the app to `jsk_robot_startup` and use the same app for `fetch`, `pr2` and other robotcs.